llvm_util/llvm2alive.cpp

```diff
@@ -1213,13 +1115,89 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
         continue;
 
       default:
-        errorAttr(attr);
-        return nullopt;
+        // If it is call site, don't raise unsupported attribute error
+        if (!is_callsite) {
```

Comment:
maybe let's change this so it never errors out.
Fncall should be tagged as approximated when an unknown attribute is used.

Reply:
I updated this part, but the approximation reason should be passed?
Since I'm not familiar with the approximation part, I made the approx flag be passed and that's it.

Reply:
the approximation code allow us to give a reason, yes. But the fncall instruction doesn't store a reason.
I think your code looks good as it is 😉